comp/otelcol/converter/README.md

```diff
@@ -24,8 +24,32 @@ If it finds datadogexporters which are not defined in a pipeline with the promet
 
 For any prometheus receiver collecting collector health metrics, and sending these to Datadog, it will update the job name to `datadog-agent`. This ensures the health metrics are tagged by `service:datadog-agent` and differentiable from collector health metrics.
 
+### Datadog Connector
+
+The converter will automatically set `datadogconnector` config `trace.span_name_as_resource_name` to true in any datadog connectors in your configuration.
+
 ## Provided and enhanced config
 
 `GetProvidedConf` and `GetEnhancedConf` return the string representation of the user provided and autoconfigured conf respectively. Currently, these APIs have two limitations:
 - They do not redact sensitive data
 - They do not provide the effective config (including defaults...etc)
+
+## Opting out of converter
+
+It is possible to opt out of the converter by setting env var `DD_OTELCOLLECTOR_CONVERTER_ENABLED` or agent config `otelcollector.converter.enabled` to `false` (`true` by default). Please note that by doing so, you are removing functionality including flare collection from otel-agent, health metrics from collector, or infra level tagging on your telemetry data. If you want to opt out of some components, you can disable all and add the components that you require manually:
+
+### Extensions
```

Comment:
Can you also add a note in the infra attrs section saying what that processor does